src/pwr_mgmt/pwr.go

```diff
@@ -27,6 +36,28 @@ func (w ReaderWriter) WriteScalingGov(sclgov string, cpu int) error {
 	return nil
 }
 
+func (w ReaderWriter) WriteCPUFreq(freqMin, freqMax, cpu int) error {
+
+	// If min frequency is set to 0 or -1, it means no limit was set
+	if freqMin != 0 && freqMin != -1 {
```

Comment:
Why two ways to check the same thing? When the value isn't set, this function should receive -1.
Value of 0 is a valid input.
```suggestion
	// If min frequency is set -1, it means no limit was set
	if freqMin != -1 {
```

Reply:
This was due to the early commits, also I wasn't considering setting the minimum freq to 0Hz was a valid thing.

Check dropped here: cc7c6d9